stackslib/src/chainstate/nakamoto/miner.rs

```diff
@@ -150,6 +150,14 @@ pub struct MinerTenureInfo<'a> {
     pub tenure_block_commit_opt: Option<LeaderBlockCommitOp>,
 }
 
+pub struct BlockMetadata {
```

Comment:
Please add docstrings for these fields

Reply:
Added in 6fccd18d9dfeccf1035d1b48a66062e4b57f3132